components/common/accordion/accordion_item.tsx

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React, {useEffect, useRef, useState} from 'react';
+
+import {AccordionData} from './accordion';
+
+import './accordion.scss';
+
+type Props = {
+    data: AccordionData;
+    isOpen: boolean;
+    btnOnClick: () => void;
+    openMultiple: boolean;
+    headerClick: <T>(...args: T[]) => T | void;
+}
+
+const AccordionItem = ({
+    data,
+    isOpen,
+    btnOnClick,
+    openMultiple,
+    headerClick,
+}: Props): JSX.Element | null => {
+    const contentRef = useRef<HTMLDivElement>(null);
+    const itemRef = useRef<HTMLDivElement>(null);
+
+    const [height, setHeight] = useState(0);
+
+    const [open, setOpen] = React.useState(false);
```

Comment:
```suggestion
    const [open, setOpen] = useState(false);
```

Reply:
for mocking the state values, it is necessary to use React.useState. It will allow you to mock the useState return using jest without using the hooks-testing-framework.